notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1623 @@
+{
```

Comment:
<p>For the <code>is_in_manhattan</code> UDF calls here, instead of making a new column and then running <code>where</code>, can you instead run <code>where</code> but filtering on the output of the UDF directly?    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>